README.rst

```diff
@@ -18,7 +18,7 @@ This product was developed for use with Plone and Dexterity.
 Requirements
 ------------
 
-* Plone >= 4.3
+* Versions >= 1.3 are for Plone 5+, if you use Plone 4.3, use versions 1.2.x
```

Comment:
It would be nice to have a
```
Products.CMFPlone >=5.0
```
in install_requires.

Reply:
done